src/lib/db/migrations/20230313130941_attachments-table.ts

```diff
@@ -0,0 +1,20 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  const [extInstalled] = await knex('pg_extension').select('*').where({ extname: 'uuid-ossp' })
+
+  if (!extInstalled) await knex.raw('CREATE EXTENSION "uuid-ossp"')
+
+  await knex.schema.createTable('attachments', (def) => {
+    def.uuid('id').defaultTo(knex.raw('uuid_generate_v4()'))
+    def.string('filename', 255).notNullable()
+    def.binary('binary_blob').notNullable()
+    def.datetime('created_at').notNullable().defaultTo(knex.fn.now())
+
+    def.primary(['id'])
+  })
+}
+
+export async function down(knex: Knex): Promise<void> {
+  await knex.schema.dropTable('attachments')
```

Comment:
Drop the extension as well.

Reply:
@dblane-digicatapult it has been updated.